src/common/Filters.js

```diff
@@ -1,3 +1,55 @@
+import { parse } from "query-string";
+
+const clearFilter = (x) => {
+  x.options.map((y) => {
+    y.checked = false;
+    return y;
+  });
+  return x;
+};
+
+/**
+ * Update filters based on a given querystring
+ * @param {array} filters list of filters in the standard format for this app
+ * @param {string} queryString querystring to be parsed
+ */
+const UpdateFilters = (filters = [], queryString = "") => {
```

Comment:
since this is pretty big, but be worth refactoring into a couple smaller functions.